README.md

```diff
@@ -129,70 +133,6 @@ You can then upload the `manylinux2014` wheels to pypi using [twine](https://git
 
 It is possible to use any of the `manylinux` docker images: `manylinux1`, `manylinux2010` or `manylinux2014`. (Just replace `manylinux2014` in the above instructions with the alternative version you wish to use.)
 
```

Comment:
I think we should add a section about how to build macOS universal2 binary wheels here.

Reply:
Agreed, I'll add it later today in a follow-up PR 👍